flink-end-to-end-tests/test-scripts/test_kubernetes_pyflink_application.sh

```diff
@@ -80,7 +80,9 @@ cp "${FLINK_PYTHON_DIR}/pyproject.toml" $PYFLINK_DOCKER_DIR/
 if [[ -d "dist" ]]; then rm -Rf dist; fi
 cd ${PYFLINK_DOCKER_DIR}
 echo "FROM ${PURE_FLINK_IMAGE_NAME}" >> Dockerfile
+echo "USER root"
```

Comment:
It would be useful to include your diagnosis of the problem and the description of the proposed change in either the Flink Jira or the PR words.